src/scripts.js

```diff
@@ -105,19 +114,19 @@ function getInstructionsElement(e, selectedRecipe) {
   }, '');
 
   return '<p class="selected-recipe-instructions">' + instructionsText + '</p>'
-};
+}
 
 function show(elements) {
   for (var i = 0; i < elements.length; i++) {
     elements[i].classList.remove('hidden');
-  };
-};
+  }
+}
 
 function hide(elements) {
   for (var i = 0; i < elements.length; i++) {
     elements[i].classList.add('hidden');
-  };
-};
+  }
```

Comment:
It's a battle of the semicolons! Some group-mates are adding and some are deleting. Before our project is finalized, we will run the linter and be consistent. :)

Reply:
I believe we decided that we want to use semi-colons! Either way, let's just make sure we're consistent. The rest of the project uses semi-colons thus far, so it is probably easiest to continue to use them. Not a big deal! Just trying to avoid some tedious refactoring down the road if we can help it